src/main/java/com/katsudon/binder/service/CardService.java

```diff
@@ -13,4 +18,13 @@ public class CardService {
   public Card create(Card card) {
     return cardRepository.save(card);
   }
+
+  public List<Card> find(List<String> rarity, List<String> set, String name,
+                         String orderBy, String orderDir, int page, int pageSize) {
+    boolean ascending = orderDir.compareTo("asc") > 0;
+    Sort sort = ascending ? Sort.by(orderBy).ascending() : Sort.by(orderBy).descending();
+    Pageable pageable = PageRequest.of(page, pageSize, sort);
+
+    return cardRepository.findByRarityInAndSetInAndName(rarity, set, name, pageable);
```

Comment:
can we make sure this is behaving as expected?
if I call
```java
cardRepository.findByRarityInAndSetInAndName(List.of("Illustration Rare"), List.of(), null, pageable)
```

does this still return all Illustration Rare cards? Can you confirm that mongodb will ignore the $in query if a list is empty?